commit 9d8980deec64c0ffebc8bd9b8504d0406b11600b
Author: Peter Hartmann <phartmann@rim.com>
Date:   Tue Nov 20 19:18:45 2012 +0100

    auto tests: only build help autotests when not cross-compiling
    
    ... because QtHelp and all other tools are only built when not cross-
    compiling.
    
    In Qt5, this is already fixed in the qttools repo.
    
    Change-Id: I151f8b4abe129def9f5f3717e72dea7dbd31fd73
    Reviewed-by: Friedemann Kleint <Friedemann.Kleint@digia.com>

diff --git a/tests/auto/auto.pro b/tests/auto/auto.pro
index 161de03..2332f8f 100644
--- a/tests/auto/auto.pro
+++ b/tests/auto/auto.pro
@@ -19,5 +19,5 @@ contains(QT_CONFIG, multimedia):            SUBDIRS += multimedia.pro
 contains(QT_CONFIG, phonon):                SUBDIRS += phonon.pro
 contains(QT_CONFIG, svg):                   SUBDIRS += svg.pro
 contains(QT_CONFIG, declarative):           SUBDIRS += declarative.pro
-!symbian:                                   SUBDIRS += help.pro
+!cross_compile:                             SUBDIRS += help.pro
 
